src/mame/tangerine/oric.cpp

```diff
@@ -37,6 +37,7 @@
 #include "emupal.h"
 #include "screen.h"
 #include "speaker.h"
+#include "softlist_dev.h"
```

Comment:
Please put the headers from src/emu in alphabetical order here.  It’s easier to keep track of them that way.

Reply:
Done